niworkflows/interfaces/masks.py

```diff
@@ -267,7 +267,7 @@ def _generate_report(self):
             nsegs = len(levels)
             if nsegs == 0:
                 levels = np.unique(np.round(
-                    nb.load(seg_files[0]).get_data()).astype(int))
+                    nb.load(seg_files[0]).get_fdata()).astype(int))
```

Comment:
Coercing to `int` anyway. Also guessing we don't need 64-bit ints...

```suggestion
                    np.asanyarray(nb.load(seg_files[0]).dataobj).astype('int32'))
```